src/js/msp/MSPHelper.js

```
@@ -2633,7 +2636,12 @@ MspHelper.prototype.sendAdjustmentRanges = function (onCompleteCallback) {
             .push8((adjustmentRange.range.start - 900) / 25)
             .push8((adjustmentRange.range.end - 900) / 25)
             .push8(adjustmentRange.adjustmentFunction)
-            .push8(adjustmentRange.auxSwitchChannelIndex);
+            .push8(adjustmentRange.auxSwitchChannelIndex)
+            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_48)) {
```

ESLint

[error] Expected indentation of 8 spaces but found 12.

(indent)